esp_idf/spotflow/srcs/logging/spotflow_log_backend.c

```diff
@@ -0,0 +1,60 @@
+#include "logging/spotflow_log_backend.h"
+
+static const char *TAG = "spotflow_testing";
+#include <stdarg.h>
+
+int spotflow_log_backend(const char *fmt, va_list args)
+{
+    // Format the log string into a buffer
+    char buffer[CONFIG_SPOTFLOW_LOG_BUFFER_SIZE];
```

Comment:
Is it safe to allocate 512 bytes (or more if configured) on the stack and not on the heap?